File: Pset1/Cash/cash.c
#include <stdio.h>
#include <cs50.h>

int main(){
    int cantidad = 0;
    do{
        cantidad = get_int("Change owed: ");
    }while(cantidad < 1);
    int conteo = 0;
    while (cantidad !=0){
        if (cantidad >= 25){
            cantidad -=25;
        }
        else if (cantidad >=10){
            
            cantidad -=10;
        }
        else if (cantidad >=5){
            cantidad -=5;
        }
        else{
            cantidad --;
        }
        conteo++;
    }
    printf("%i\n",conteo);
    return 0;
}

File: Pset6/Sentimental-Cash/cash.py
from cs50 import get_float

cantidad:float = 0

while True:
    cantidad = get_float("Change owed: ")
    if cantidad >=0: break
cambio:int = 0

while cantidad >0:
    if cantidad >=0.25:
        cantidad -=0.25

    elif cantidad >=0.10:
        cantidad -=0.10

    elif cantidad >=0.05:
        cantidad -=0.05

    else:
        cantidad -=0.01
    cantidad = round(cantidad,2)
    cambio+=1

print(cambio)
